lib/ugrid_checks/check.py

```diff
@@ -1790,64 +1812,125 @@ def report(self, include_nonmesh: bool = False) -> str:
             self._nonmesh_report()
         return "\n".join(self.result_lines)
 
+    def _valid_var(self, var_name: str):
+        # Return var or None
+        return self.all_file_vars.get(str(var_name))
+
+    def _valid_vars_map(self, var_names: List[str]):
+        # Return dict of {name:var}, only valid vars
+        varsdict = {}
+        for name in var_names:
+            var = self.all_file_vars.get(str(name))
+            if name is not None:
+                varsdict[name] = var
+        return varsdict
+
     def _mesh_report(self):
         if not self.mesh_vars:
             self._line("Meshes : <none>")
         else:
             self._line("Meshes")
             for mesh_name, mesh_var in self.mesh_vars.items():
+                struct_mesh = UgridMesh(name=mesh_name, var=mesh_var)
```

Comment:
This whole approach is a preliminary hack, for simple concept testing only.
Proper approach TBD